presto-docs/src/main/sphinx/release/release-0.183.rst

```diff
@@ -0,0 +1,61 @@
+=============
+Release 0.183
+=============
+
+General Changes
+---------------
+
+* Fix planning failure for queries that use ``GROUPING`` and contain aggregation expressions
+  that require implicit coercions.
+* Fix planning failure for queries that contains a non-equi left ``JOIN`` that is semantically
```

Comment:
Remove the code formatting here since this is an English description rather than referring to specific syntax. We'd need to either say "LEFT JOIN" or "left join". Splitting is weird.

Similarly, we could replace "GROUPING" in the above with "grouping sets". But that one is fine either way.